src/css/top-section.css

```diff
@@ -56,7 +56,6 @@
 .mobileLinkText {
   text-decoration: none;
   color: #fff5e1;
-  font-family: 'Inter', serif;
```

Comment:
- [ Optional ] As stated by the previous reviewer Kindly use` font-family: 'Cocogoose', sans-serif;`or `font-family: 'Lato', sans-serif;` to stay consistent in your design 👍 .